webapp/src/js/spotboard.js

```diff
@@ -43,7 +43,7 @@ function(jQuery, __meta__) {
         if(rm) config['animation'] = (rm[1] == 'true');
 
         rm = /q=([-*+% \w]+)/.exec(wls);
-        if(rm) config['search_query'] = '^' + decodeURIComponent(rm[1]) + '$';
+        if(rm) config['search_query'] = decodeURIComponent(rm[1]);
```

Comment:
why this? It was intentional because some school name can be a prefix of other.

Reply:
Still they can use regex in search query. e.g. searhcing with "korea university$"

Reply:
Without this change, it doesn't work well. Because this PR remember searching query with URL parameter.